code/drasil-theory/lib/Theory/Drasil/DataDefinition.hs

```diff
@@ -108,23 +109,26 @@ instance RequiresChecking DataDefinition Expr Space where
 
 -- * Constructors
 
+ddUid :: HasUID c => c -> UID
+ddUid = nsUid "dataDefn" . (^. uid)
```

Comment:
1. Here we are moving DataDefinitions to the "dataDefn" namespace.

I don't think this is the right naming scheme ultimately. I want to see `UID`s rethought later, but for now, this at least lets us remove the UID duplication between `dataDefn`s and everything else but the term cache.

Reply:
agree that this is likely temporary, but useful nevertheless.